<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="login" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>719</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QRadioButton" name="M_rbtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>430</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="F_rbtn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>430</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cama_le">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>315</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="last_name_le">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>253</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edad_le">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>191</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="sala_le">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>130</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="municipio_le">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>69</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="first_name_le">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>375</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ID_le">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>8</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ok_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_btn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>480</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
